tealeaf/platform/PluginManager.h

```diff
@@ -40,6 +40,7 @@
 - (void) onPause;
```

Comment:
nope. I checked. on opening our app from link, this callback is not getting triggered.